File: Source/Front-end/front-end/src/app/model/response/get-residenti-response.ts
export class GetResidentiResponse {
  email: string;
  nome: string;
  cognome: string;
  stanza: string;
  dataDiNascita: Date;

  constructor(email: string, nome: string, cognome: string, stanza: string, dataDiNascita: Date) {
    this.email = email;
    this.nome = nome;
    this.cognome = cognome;
    this.stanza = stanza;
    this.dataDiNascita = dataDiNascita;
  }
}

File: Source/Back-end/src/main/java/org/elis/manutenzione/dto/response/GetResidentiResponse.java
package org.elis.manutenzione.dto.response;

import lombok.Data;

import java.time.LocalDate;

/**
 * DTO che rappresenta la risposta per la richiesta di ottenere i dati dei residenti.
 *
 */
@Data
public class GetResidentiResponse {

    /**
     * Il nome del residente.
     */
    private String nome;

    /**
     * Il cognome del residente.
     */
    private String cognome;

    /**
     * Il nome della stanza assegnata al residente.
     */
    private String stanza;

    /**
     * L'indirizzo email del residente.
     */
    private String email;

    /**
     * La data di nascita del residente.
     */
    private LocalDate dataDiNascita;
}